### PR TITLE
Test for independent drag of nested draggable objects

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by jbalsas** [comment]
Hey @Robert-Frampton, this addresses https://github.com/metal/metal-drag-drop/issues/20. Looks good to me, but since I worked on it with @pavel-savinov I'd appreciate an extra pair of eyes 😉

File: src/Drag.js [hunk 1]
```diff
@@ -462,6 +462,8 @@ class Drag extends State {
 	 * @protected
 	 */
 	handleDragStartEvent_(event) {
+		event.stopPropagation();
```
Comment (Robert-Frampton) [inline src/Drag.js]: I see that `preventDefault` is also called in this method, but only when `canStartDrag_` returns true. Can we do that same for `stopPropagation`?

Comment (pavel-savinov) [reply src/Drag.js]: It will cause overwriting of this.activeDragSource_ and it is exactly what we want to avoid.

Comment (jbalsas) [reply src/Drag.js]: Maybe we can move the `activeDragSource_` inside the if as well?

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/metal/metal-drag-drop/pull/22. [See changes here.](https://github.com/metal/metal-drag-drop/compare/6e41007~1...6e41007) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**